### ISSUE TITLE
VS Code title is completely invisible 

### ISSUE THREAD

**Issue by cessez-le-feu** [barjoio/anirak#2]
Hello,

Discovered your VS Code theme today. Looks absolutely stunning, both in the Light and Dark versions. But in either version, VS Code title is completely invisible. I don’t have this problem with other themes, and for debugging purposes, I ran the VS Code command “Developer: Generate Color Theme From Current Settings”:

```
...
"titleBar.activeBackground": "#ecedf4",
"titleBar.activeForeground": "#ecedf4",
...
"titleBar.inactiveBackground": "#ecedf4",
"titleBar.inactiveForeground": "#ecedf4",
...
```

So, you are using the same background and foreground colours for the title. Is this deliberate or not? If it is, it would be better to provide some sort of override option.

**Comment by barjoio** [comment]
Thanks for the kind words and I'm glad you're enjoying the theme. I appreciate you pointing out this problem, it has now been fixed! An update has already rolled out so you should see the changes very soon.

Please do not hesitate to open another issue if you spot anything else wrong with the theme.

**Comment by cessez-le-feu** [comment]
Thank you for your very prompt and effective reaction! I confirm that the title issue is fixed in 6.1.0.
